vllm/model_executor/models/arctic.py

```diff
@@ -360,6 +361,7 @@ def forward(
         return hidden_states
 
 
+@support_torch_compile
 class ArcticModel(nn.Module):
```

Comment:
to run this model successfully on H100, I have to change the config:

```text
"hidden_size": 512,
"intermediate_size": 512,
"num_key_value_heads": 8,
"num_attention_heads": 8,
"num_local_experts": 4,
```

initially, I want to simply change `"num_hidden_layers": 35,` to `"num_hidden_layers": 2,` , but I met various random illegal memory access error. might be caused by fused moe kernel, with extremely large input sizes.